src/test/kotlin/no/nav/syfo/dialogmote/api/v2/FerdigstillDialogmoteApiV2Spek.kt

```diff
@@ -346,6 +346,134 @@ class FerdigstillDialogmoteApiV2Spek : Spek({
                         }
                     }
                 }
+                describe("Happy path: with behandler og mellomlagring") {
```

Comment:
Det er mange steg her for å komme til mellomlagring av referat, og mye går vel igjen i alle testene? Er det mulig å skille ut de ulike delene - innkalling, hente møte osv. - i ulike metoder, så hver test blir litt mindre og mer oversiktlig?

Reply:
Gjør et forsøk på en refaktorering her i en egen PR etterpå.